src/classes/LineItemTriggerHandler.cls

```diff
@@ -36,6 +37,7 @@ public with sharing class LineItemTriggerHandler {
 			thisOrderAmount = (thisOrderAmount!=null) ? thisOrderAmount : 0; 
```

Comment:
This data manipulation is inside a loop. Please use a collection to store relevant data, and move any database manipulation outside the loop.